src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -379,6 +370,20 @@ private Container createContainer(KubernetesSlave slave, ContainerTemplate conta
             containerMounts.add(new VolumeMount(containerTemplate.getWorkingDir(), WORKSPACE_VOLUME_NAME, false, null));
         }
 
+        Probe livenessProbe = null;
+        for (ContainerLivenessProbe clp : containerTemplate.getLivenessProbe()) {
+            if (parseLivenessProbe(clp.getExecArgs()) != null) {
+                livenessProbe = new ProbeBuilder()
```

Comment:
you have a list of probes and then you just overwrite all of them but the latest one
should containerTemplate just have one `livenessProbe` ?